### ISSUE TITLE
Accessibility for Calendar component

### ISSUE THREAD

**Issue by mohammedzamakhan** [momentum-design/momentum-ui#93]
## Issue Type
Bug Report

## Component or Pattern Affected
Date Picker

## Expected Behavior
The voiceover/screen reader should be able to speak out the dates when using the keyboard to select dates, similarly to how it works for https://material.angular.io/components/datepicker/overview

## Current Behavior
While using voiceover, and then using keys to select dates, I did not here the voiceover informing the date the current selection is on

## Possible Solution
Similar to how material angular does it

## Steps to Reproduce (for bugs)

1. 
1. 
1. 
1. 

## Your Environment

* Library: 
* Version used: 
* Browser Name and version: 
* Operating System and version: 
* Link to your project:

**Comment by bfbiggs** [comment]
Closing as angular library has been forked and removed.
